geoportailv3/static/js/profile/profiledirective.js

```diff
@@ -0,0 +1,237 @@
+/**
+ * @fileoverview This file provides a measure directive. This directive is used
+ * to create a measure panel in the page.
+ *
+ * Example:
+ *
+ * <app-measure app-measure-map="::mainCtrl.map"></app-measure>
```

Comment:
The example refers to the `app-measure` directive, which is not the directive this file includes.